tests/integration/test_gardenlinux.py

```diff
@@ -10,15 +10,15 @@
 import pytest
 import yaml
 
-from .aws import AWS
-from .gcp import GCP
-from .azure import AZURE
-from .openstackccee import OpenStackCCEE
-from .chroot import CHROOT
-from .kvm import KVM
-from .manual import Manual
-from .ali import ALI
-from .sshclient import RemoteClient
+from integration.aws import AWS
+from integration.gcp import GCP
+from integration.azure import AZURE
+from integration.openstackccee import OpenStackCCEE
+from integration.chroot import CHROOT
+from integration.kvm import KVM
+from integration.manual import Manual
+from integration.ali import ALI
+from integration.sshclient import RemoteClient
```

Comment:
Would it make sense to move `integration.sshclient` to `helper.sshclient`?
This way, we could probably call `integration` `platforms` but we should discuss this with the team first.

Reply:
As discussed, we would create a dedicacted issue for this since it isn't directly linked to the issue this PR is resolving.